wcc-contentful-app/lib/wcc/contentful/app/middleware/publish_at.rb

```diff
@@ -4,9 +4,38 @@ module WCC::Contentful::App::Middleware
   class PublishAt
     include WCC::Contentful::Middleware::Store
 
+    class << self
+      def job_entry_storage
+        @job_entry_storage ||= Redis::Store.new
+      end
+      attr_writer :job_entry_storage
+
+      def entry_key(entry)
+        [name, entry.dig('sys', 'type'), entry.dig('sys', 'id')].join('.')
+      end
+
+      def update_storage(entry, force = false)
+        key = entry_key(entry)
+        return job_entry_storage.set(key, entry) if force
+
+        return unless old_entry = job_entry_storage.get(key)
+        return unless old_revision = old_entry.dig('sys', 'revision')
+        return unless old_revision < entry.dig('sys', 'revision')
+
+        job_entry_storage.set(key, entry)
+      end
+
+      def latest_entry_version?(entry)
+        # If the entry isn't in the job storage, then something's changed.
+        return false unless from_storage = job_entry_storage.get(entry_key(entry))
+
+        entry.dig('sys', 'revision') >= from_storage.dig('sys', 'revision')
+      end
+    end
+
     def self.call(store, content_delivery_params, config)
       # This does not apply in preview mode
-      return if content_delivery_params&.find { |array| array[:preview] == true }
+      return if content_delivery_params&.find { |h| h.is_a?(Hash) && (h[:preview] == true) }
```

Comment:
what would `h[:preview]` be otherwise?

Reply:
h could be a string, or something else that doesn't respond to `:[]`.

Reply:
In wm-resources
```
config.content_delivery = :lazy_sync, Rails.cache if Rails.env.production?
```

Reply:
My question centered around `== true` instead of just truthiness